chatbot/db/db.js

```diff
@@ -11,7 +11,19 @@ pool.on('error', (err) => {
     console.error('unexpected database error:', err)
 })
 
-function createSessionFromRow(r) {
+async function createSessionFromRow(client, r) {
+
+    if(!r) {
+        return null
+    }
+
+    let { rows } = await client.query("SELECT * FROM installations WHERE id = $1", [r.installation_id])
+    let className = rows[0].session_state_class_name
```

Comment:
throwing an "installation not found (id=x)" error would make it easier to diagnose the cause if this receives an invalid installation.